docs/reference/inference/service-elasticsearch.asciidoc

```diff
@@ -0,0 +1,112 @@
+[[infer-service-elasticsearch]]
+=== Elasticsearch {infer} `service`
+
+Creates an {infer} endpoint to perform an {infer} task with the `elasticsearch` service.
+
+
+[discrete]
+[[infer-service-elasticsearch-api-request]]
+==== {api-request-title}
+
+`PUT /_inference/<task_type>/<inference_id>`
+
+[discrete]
+[[infer-service-elasticsearch-api-path-params]]
+==== {api-path-parms-title}
+
+`<inference_id>`::
+(Required, string)
+include::inference-shared.asciidoc[tag=inference-id]
+
+`<task_type>`::
+(Required, string)
+include::inference-shared.asciidoc[tag=task-type]
++
+--
+Available task types:
+* `text_embedding`.
+--
+
+[discrete]
+[[infer-service-elasticsearch-api-request-body]]
+==== {api-request-body-title}
+
+`service`::
+(Required, string)
+The type of service supported for the specified task type. In this case, 
+`elasticsearch`.
+
+`service_settings`::
+(Required, object)
+include::inference-shared.asciidoc[tag=service-settings]
++
+--
+These settings are specific to the `service` you specified which is `elasticsearch` in this case.
+--
+
+`model_id`:::
+(Required, string)
+The name of the model to use for the {infer} task.
+It can be the ID of either a built-in model (for example, `.multilingual-e5-small` for E5) or a text embedding model already
+{ml-docs}/ml-nlp-import-model.html#ml-nlp-import-script[uploaded through Eland].
+
+`num_allocations`:::
+(Required, integer)
+The number of model allocations to create. `num_allocations` must not exceed the number of available processors per node divided by the `num_threads`.
+
+`num_threads`:::
+(Required, integer)
+The number of threads to use by each model allocation. `num_threads` must not exceed the number of available processors per node divided by the number of allocations.
+Must be a power of 2. Max allowed value is 32.
+
+`task_settings`::
+(Optional, object)
+include::inference-shared.asciidoc[tag=task-settings]
```

Comment:
Doesn't look like this is importing anything.

The rerank specific task settings are:

```
{
  return_documents: true|false. Defaults to true. Returns the document instead of only the index.
}
```

There aren't any task settings for `text_embedding`.